main.js

```diff
@@ -0,0 +1,243 @@
+//  ----------------VARIABLES--------------------------------//
+
+const nav = document.querySelector('.menu');
+const menubutton = document.querySelectorAll('#menubutton');
+const iconsButtons = document.querySelectorAll('.menu-links');
+const closeButtom = document.querySelectorAll('.close-buttom');
+
+//  -----------------Hamburger menu--------------------------//
+
+function toggleMenu() {
+  nav.classList.toggle('active');
+}
+
+function closeMenu() {
+  nav.classList.remove('active');
+}
+
+menubutton.onclick = () => { toggleMenu(); };
+iconsButtons.onclick = () => { closeMenu(); };
+
+//  -------------------Modal functions OPEN/CLOSE-----------------//
+
+function toggleModal(ID) {
+  const modal = document.getElementById(ID);
+  modal.classList.add('showUp');
+
+  window.onclick = function (event) {
+    if (event.target === modal || event.target.className === 'close-buttom') {
+      modal.classList.remove('showUp');
+    }
+  };
+
+  closeButtom.onclick = function () {
+    modal.classList.remove('showUp');
+  };
+}
+
+//  -------------------DATA STORAGE (array of objects)------------------------//
+
+//  CARDS DATA
+const projects = [
+  {
+    projectID: 0,
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    featuredImageBig: 'images/firstprojectbig.png',
+    featuredImageSmall: 'images/backgroundprojects2.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    linkLiveVersion: 'https://www.google.com',
```

Comment:
**[REQUIRED CHANGES]**

- Hey, please kindly ensure that the link to  **all the live version links** takes a user to any of your live demo links to your project on Github.

**This change applies to all the project cards**

Reply:
@zieeco Hello, my friend! I just talked to the student success about this issue since I'm on the first module and I still don't have any demo links so they told me that I shouldn't care about this one for now since it's not on the requirements. I'll make all the other changes as soon as possible. Thank you so much for the advices.

Reply:
@VitorGuedesMadeira,
You can skip this one for now, but ensure you fix the rest issue and submit for a re-review. 
Cheers and Happy Coding!